packages/starknet/lib/src/account.dart

```diff
@@ -161,6 +166,16 @@ class Account {
     return txHash;
   }
 
+  Future<bool> get isValid async {
```

Comment:
`isValid` seems a bit generic.
Maybe comment or rename to better explain what is a "valid" or "invalid" account ?

Reply:
A comment have been added.
I'm not sure if we should call **valid** or **deployed**.